setup-Hhh/tt/cgs-Hhh-8TeV-00.conf

```diff
@@ -1,6 +1,6 @@
 # cgs.config: Specification of groups, categories, and samples
 $ GROUP signal ggHTohhTo2Tau2B
-$ GROUP background ZTT,QCD,W,TT,VV,ZJ
+$ GROUP background ZTT,QCD,TT,VV,ZLL,W
```

Comment:
Was it intended to keep W+jets only in this category? - it no longer has any uncertainties assigned in the .vals file